workspaceitem-data-upload.md

```diff
@@ -41,7 +43,9 @@ The section data represent the data about the user uploaded files
  	]
 }
 ```
-the files attribute contains the list of user uploaded file in the section. For each file the following attributes exist
+the primary attribute contains eventually the uuid of the bitstream set as primary, it will be null if no primary bitstream is set for the ORIGINAL bundle.
+The files attribute contains the list of user uploaded file in the section. For each file the following attributes exist
+* id: the uuid of the underline bitstream. Useful to set the primary attribute
```

Comment:
do we mean "underlying" here?

Reply:
@hardyoyo is right.  This shouldn't say "underline bitstream".  Instead it should either say "underlying bitstream" or just "bitstream" works too.